.github/workflows/approve-command.yml

```diff
@@ -0,0 +1,132 @@
+name: Approve Command
+
+on:
+  issue_comment:
```

Comment:
not sure if issue_comment is right. I gotta check if we need something specific to PRs instead.

Reply:
I don't see any action added in the CI list or any new action job in the Actions tab of the repo. Perhaps the `on:` section needs a `pull_request` entry like: https://github.com/marketplace/actions/comment-pull-request#classic-usage